DEPENDENCIES

```diff
@@ -976,22 +964,21 @@ npm/npmjs/-/yocto-queue/0.1.0, MIT, approved, clearlydefined
 npm/npmjs/@adobe/css-tools/4.0.1, MIT, approved, #6345
 npm/npmjs/@ampproject/remapping/2.2.0, Apache-2.0, approved, clearlydefined
 npm/npmjs/@apideck/better-ajv-errors/0.3.6, MIT, approved, clearlydefined
-npm/npmjs/@babel/code-frame/7.12.11, MIT, approved, clearlydefined
-npm/npmjs/@babel/code-frame/7.18.6, MIT, approved, clearlydefined
+npm/npmjs/@babel/code-frame/7.23.4, , restricted, clearlydefined
```

Comment:
Please update `DEPENDENCIES` file after successfull IP-Check for restricted 3rd party libs.